library/consumer-proguard-rules.pro

```diff
@@ -5,4 +5,4 @@
   @com.google.gson.annotations.SerializedName <fields>;
 }
 
--keepclassmembers enum com.wultra.android.mtokensdk.api.*.** { *; }
\ No newline at end of file
+-keep, allowobfuscation class com.wultra.android.mtokensdk.api.** { *; }
```

Comment:
If it's for Gson, you probably need only `{ <fields>; }` inside (there are a lot of generated Kotlin methods that would be kept otherwise).
Maybe something like this might be good:
```
# handle Gson
-keepclassmembers class com.wultra.android.mtokensdk.api.** {
     <fields>;
}
# handle R8 full mode optimizations
-keep, allowobfuscation class com.wultra.android.mtokensdk.api.**
```

Moreover if it is for Gson, then you are keeping a ton of unnecessary classes. You need this only for the model classes.  Only those are affected by R8 in full mode.